+nosnoc/+dcs/Objects.m

```diff
@@ -0,0 +1,128 @@
+classdef Objects < nosnoc.dcs.Base
+% A particular class of dynamic complementarity system called a gradient complementarity system.
```

Comment:
this is the description of the GCS class. probably a copy past leftover, please provide better description